abe/sample_data.py

```diff
@@ -230,7 +88,6 @@ def load_data(
         for event in event_data:
             for key, value in event.items():
                 if type(value) is datetime:  # convert to UTC from EST
-                    og = value
```

Comment:
Who's og, anyway!? (Was causing a linter warning.)

Reply:
I'm pretty sure that dangling initialization is my fault.